rootfs/api/models/config.py

```diff
@@ -105,6 +111,40 @@ def set_tags(self, previous_config):
 
         raise DeisException(message)
 
+    def set_healthcheck(self, previous_config):
+        data = getattr(previous_config, 'healthcheck', {}).copy()
+        new_data = getattr(self, 'healthcheck', {}).copy()
+        # update the config data for healthcheck if they are not
+        # present for per proctype
+        if 'livenessProbe' in data.keys() or 'readinessProbe' in data.keys():
```

Comment:
the function gets called but the if doesn't pass as keys can't be probes unless it is read from the database.